homework/index.js

```diff
@@ -43,8 +77,13 @@
         });
         return;
       }
-      const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      createAndAppend('div', root, {
+        id: 'title',
+        text: 'HYF Repositories',
+      });
+      createAndAppend('ul', root, { id: 'reposList' });
+      sortRepos(repos);
+      repos.forEach(repo => renderRepoDetails(repo));
```

Comment:
You do the sort in-line and then chain with `forEach()`:

```js
repos
  .sort((a, b) => a.name.localeCompare(b.name));
  .forEach(repo => ...);
```

Reply:
I did not quite understand this remark. Do you mean that It does not need to make a separate function for sorting and just sort in a line is enough?

Reply:
I have the unfortunate habit of sometimes leaving out words when I type. This is a typical case:

> You **can** do the sort in-line and then chain with `.forEach()`:

Reply:
Ah, ok, I got it.